tests/ops/op_math/test_sum.py

```diff
@@ -273,6 +273,17 @@ def test_terms_mixed(self, op, coeffs_true, ops_true):
         assert coeffs == coeffs_true
         assert ops1 == ops_true
 
+    @pytest.mark.parametrize(
+        "op, coeffs_true, ops_true", SUM_TERMS_OP_PAIRS_PAULI + SUM_TERMS_OP_PAIRS_MIXEDPAULI
+    )
+    def test_terms_does_not_change_queue(self, op, coeffs_true, ops_true):
+        """Test that calling Prod.terms does not queue anything."""
+        with qml.queuing.AnnotatedQueue() as q:
+            qml.apply(op)
+            _, _ = op.terms()
+
+        assert q.queue == [op]
+
```

Comment:
I guess this is technically a separate bug fix? Might be worth highlighting in the `changelog/bugfix` section

Reply:
I actually added this fix to the0.35 RC branch, but needed to have it in for this PR, so I just duplicated the change, so the changelog for this change is in 0.35.

Reply:
Ah okay so this diff will change with the release and RC/master sync then I guess?